docs/Makefile

```diff
@@ -6,7 +6,7 @@ SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = ReadtheDocsSphinxTheme
 SOURCEDIR     = .
-BUILDDIR      = _build
+BUILDDIR      = build
```

Comment:
I personally prefer _build. Why change it?

Reply:
I will open a new PR to fix this. Because this is a bigger change.